APKBUILD

```diff
@@ -47,6 +48,6 @@ i18n() {
   cp -a "$srcdir"/usr/glibc-compat/share "$subpkgdir"/usr/glibc-compat
 }
 
-md5sums="5950e0b8f0bae4af07b405c7b1f6e194  glibc-bin-2.23-0-x86_64.tar.gz
+md5sums="3f84f6dbaffacc20fb5ad5337f98dc80  glibc-bin-$_filever-x86_64.tar.gz
```

Comment:
🙋  Was this variable added by hand? The values of these variables are automatically generated by `apk checksum` and will be overwritten on the next run.